### ISSUE TITLE
Make install file compatible with Winget 

### ISSUE THREAD

**Issue by AdrianoCahete** [maphew/svg-explorer-extension#90]
[Winget](https://github.com/microsoft/winget-cli/) is the new official Microsoft Package Manager.

Packages are [here](https://github.com/microsoft/winget-pkgs).

**Comment by Kenya-West** [comment]
Or Microsoft Store!

Microsoft Store is one place for everything in your life, moving your experience to a safe one and making it accessible from many devices that added to your Microsoft account.

Publishing apps to Microsoft Store is benefit both for users and developers + enterprise. Deploying classic apps, that available in Microsoft Store, with safe methods to a verified environments is trusted by enterprises and users. From the other hand, developer can benefit from "In-App Purchase" (paid features of the app), making its own Insider programs, setting up subscriptions and advertisements.

Microsoft corporation has opened a door to developers of classic apps, the way to publish your app to Microsoft Store. UWP brings a second life for your apps: notifications, full power of trusted UWP/WinRT API, easy debug, easy deploy, additional features.

Learn more about that easy process [here](https://developer.microsoft.com/windows/bridges/desktop).

Best regards,
Innokenty Ivanov,
North-Eastern Federal University student (PhD student).
Programmer, web developer, **Microsoft Student Partner**.
My [e-mail](mailto:Kenya-West@outlook.com).

**Comment by LuisAPI** [comment]
Agreed! It seems this is actually available right now on Winget, but the only versions it could detect are previous 2 ones: 1.0.0 and 0.1.1: https://winget.run/pkg/SVGExplorerExtension/SVGExplorerExtension

### WinGet Listing Details

```
PS C:\Windows\system32> winget show svgexplorerextension.svgexplorerextension
Found SVG See [SVGExplorerExtension.SVGExplorerExtension]
Version: 1.0.0
Publisher: Tibold Kandrai
Publisher Url: https://github.com/tibold/svg-explorer-extension
Publisher Support Url: https://github.com/tibold/svg-explorer-extension/issues
Author: Tibold Kandrai
Moniker: svg-see
Description: Extension module for Windows Explorer to render SVG thumbnails, so that you can have an overview of your SVG files.
Homepage: https://github.com/tibold/svg-explorer-extension
License: GNU Lesser General Public License v3.0
License Url: https://github.com/tibold/svg-explorer-extension/blob/master/LICENSE.md
Copyright: Copyright (C) 2007 Free Software Foundation, Inc.
Copyright Url: https://github.com/tibold/svg-explorer-extension/blob/master/LICENSE.md
Tags:
  explorer
  svg
  svg-explorer-extension
  svgexplorerextension
Installer:
  Installer Type: inno
  Installer Locale: en-US
  Installer Url: https://github.com/tibold/svg-explorer-extension/releases/download/v1.1.0/svg_see_x64.exe
  Installer SHA256: 69f9721c6605b487bc78cd90439ec7b0819ad8bf77616f3cbf3b5e18b9fca6db
  Offline Distribution Supported: true
  Dependencies:
    - Package Dependencies:
        Microsoft.VCRedist.2015+.x64 [>= 14.26.28720.3]
```

**Comment by AdrianoCahete** [comment]
Just for the sake of curiosity: I'm using [MS PowerToys](https://github.com/microsoft/PowerToys) for now. It comes with the [File Explorer](https://learn.microsoft.com/en-us/windows/powertoys/file-explorer) addon that supports svg viewer, and it's constantly updated (last update here was 5y ago).

Closing because I don't use anymore (and I don't think that it will be resolved).
